lang/en.js

```diff
@@ -187,6 +187,8 @@ export default {
 	"practice:learndl:attributes": "Attribute",
 	"practice:learndl:deleteAnswer": "Delete Answer",
 	"practice:learndl:deleteAttribute": "Delete Attribute",
+	"practice:learndl:duplicateAnswer": "Duplicate answer",
+	"practice:learndl:duplicateAttribute": "Duplicate attribute",
```

Comment:
nit: for consistency, though I fully expect there to be some kind of audit on these terms down the road. 
```suggestion
	"practice:learndl:duplicateAnswer": "Duplicate Answer",
	"practice:learndl:duplicateAttribute": "Duplicate Attribute",
```